models/spring-ai-anthropic/src/main/java/org/springframework/ai/anthropic/AnthropicChatModel.java

```diff
@@ -154,29 +154,29 @@ public AnthropicChatModel(AnthropicApi anthropicApi, AnthropicChatOptions defaul
 	 * @param anthropicApi the lower-level API for the Anthropic service.
 	 * @param defaultOptions the default options used for the chat completion requests.
 	 * @param retryTemplate the retry template used to retry the Anthropic API calls.
-	 * @param functionCallbackContext the function callback context used to store the
-	 * state of the function calls.
+	 * @param functionCallbackResolver the function callback resolver used to resolve the
+	 * function by bean name
```

Comment:
Javadoc doesn't need to refer "bean" and just "by name" is enough. will fix when merging.